File: C Programming/Assignments/Assignment 3 (For loop )/Q5.c
//armstrong
void main() {
    int no=;
    int tno=no,q=0;
    for(int t1;tno>0;tno=tno/10){
        t1=tno%10;
        t1=t1*t1*t1;
        q=q+t1;
    }
    if(q==no){
        printf("The no is armstrong\n");
    }else
    {
        printf("not armstrong");
    }
}

File: Java/C TO JAVA/Assignment 3/Q5.java
class Q5
{
//armstrong
public static void main(String args[]){
    int no=123;
    int tno=no;
    int t1,q=0;
    while(no>0){
        t1=no%10;
        no=no/10;
        t1=t1*t1*t1;
        q=q+t1;
       
    }
    if(q==tno){
        System.out.printf("The no is armstrong\n");
      
    }else{
        System.out.printf("not armstrong");
    }
}
}